frontend/src/features/permits/components/permit-list/Columns.tsx

```diff
@@ -16,15 +16,15 @@ export const PermitsColumnDefinition: MRT_ColumnDef<Permit>[] = [
     enableSorting: true,
     size: 500,
     accessorFn: (row) => row.permitNumber,
-    Cell: (props: { row: any; renderedCellValue: any }) => {
+    Cell: (props: { row: any; cell: any }) => {
       return (
         <>
           <Link
             component="button"
             variant="body2"
             onClick={() => viewPermitPdf(props.row.original.permitId)}
           >
-            {props.renderedCellValue}
+            {props.cell.getValue()}
```

Comment:
The rationale for this change is that renderCellValue is an Element with some highlighting rather than the plain value of the cell. This means that if there's a match with the search string, it will get highlighted. 

The user can only can search by Unit Number and Plate and we do not want to mislead them into thinking they can search by permit number.